test/mesh_test.cpp

```diff
@@ -144,6 +144,8 @@ TEST(Manifold, GetMesh) {
   Identical(mesh_out, mesh_out2);
 }
 
+// temporarily disabled for wasm due to test failure
+#ifndef __EMSCRIPTEN__
```

Comment:
Let's just disable this one generally: rename it `DISABLED_Determinism`. It's not even consistent between CUDA and CPP, so it's unfair to blame WASM.